tools/wptserve/tests/functional/test_pipes.py

```diff
@@ -101,30 +96,27 @@ def test_sub_location(self):
 port: {0}
 query: ?query_string
 scheme: http
-server: http://localhost:{0}""".format(self.server.port)
+server: http://localhost:{0}""".format(self.server.port).encode("ascii")
```

Comment:
Is there any reason not to use a binary string here? format still works, no?

Reply:
No, `bytes` doesn't have `format` method. We could use `%`, but it'd be a bit clumsy as we need to repeat the port number a few times.